src/main/java/fr/jmmc/oiexplorer/core/gui/FitsImagePanel.java

```diff
@@ -510,6 +512,7 @@ public void itemStateChanged(java.awt.event.ItemEvent evt) {
             }
         });
         jPanelOptions.add(jToggleButtonRuler, new java.awt.GridBagConstraints());
+        jPanelOptions.add(sliderPanel, new java.awt.GridBagConstraints());
```

Comment:
No, by default do not add the sliderPanel.
You should use fitsImagePanel.addOptionPanel(sliderPanel) in the parent view